any_run/icon_any_run/util/api.py

```diff
@@ -0,0 +1,50 @@
+import requests
+from insightconnect_plugin_runtime.exceptions import PluginException
+import json
+
+
+class AnyRunAPI:
+    def __init__(self, authentication_header, logger):
+        self.url = 'https://api.any.run/v1/analysis/'
+        self.authentication_header = authentication_header
+        self.logger = logger
+
+    def get_history(self, team, skip, limit):
+        try:
+            r = requests.api.get(self.url,
+                                 params={"team": team, "skip": skip, "limit": limit},
+                                 headers=self.authentication_header)
+            r.raise_for_status()
+            return r.json()
```

Comment:
Defensive programming here, try/except to capture in case we don't get valid JSON

```
raise PluginException(preset=PluginException.Preset.INVALID_JSON, data=response.text)
```